packages/client/merge/README.md

```diff
@@ -0,0 +1,93 @@
+- Beacon chain can be followed at: https://sepolia.beaconcha.in
+
+To run `sepolia`, the configurations are baked in for both EL and CL clients. Our instructions will focus on the `lodestar` CL client, however other CL clients (`lighthouse`,`nimbus`,`teku`,`prysm`) can be similarly hooked up to `ethereumjs` as they all (should) follow `engine` API spec. 
+
+### Jwt-secret
+The EL (`ethereumjs`) and CL (`lodestar`) communications over the `engine` API are protected by `jwt` auth. To enable this, you will need to generate and provide a jwt secret. We just use here a dummy one for illustrations, please generate one of your own. 
+
+```shell
+echo "0xdc6457099f127cf0bac78de8b297df04951281909db4f58b43def7c7151e765d" > /path/to/jwtsecretfile
+```
+
+### Execution - EthereumJS Setup
+
+Please ensure you have Node 16.x installed.
+
+#### Client Installation
+
+##### GitHub
+
+1. `git clone --depth 1 --branch master https://github.com/ethereumjs/ethereumjs-monorepo.git`
+2. `cd ethereumjs-monorepo`
+3. `npm i`
+4. `cd packages/client`
+
+##### NPM
+
+```shell
+npm install -g @ethereumjs/client
+```
+
+Note that you eventually want to start the client just with `ethereumjs` instead of using the `npm run client:start` GitHub start command (also leave the in-between `--` to forward the client config options).
+
+##### Docker
+
+Or try it in Docker.
+
+In `packages/merge`, you can build a docker image using
+```shell
+docker build . --tag ethereumjs:latest
+```
+You may now do appropriate directory/file mounts for `data` dir and `jwtsecret` file and provide their path appropriately in the `client` run command.
+
+#### Run client
+
+This the the bare minimum command required for interop with the CL `lodestar`. You may add further args to expose functionality for your use case.
+
+```shell
+npm run client:start -- --network sepolia --datadir /path/to/sepolia/data/dir --rpcEngine --jwt-secret /path/to/jwtsecret/file
+```
+
+This will start `ethereumjs` client on `sepolia` network and expose `engine` endpoints at `8551`(default port, can be modified see cli help via `--help`).
+
+In case you want to host `engine_*` without auth (not recommended, only for debugging purposes), pass `--rpcEngineAuth false` as extra argument in the above run command (and you may skip `--jwt-secret`).
+
+
+### Consensus - Lodestar Setup
+
+#### Beacon
+
+1. Use lodestar branch `unstable` and run `yarn && yarn build`
+2. Run cmd: `./lodestar beacon --network sepolia --rootDir /path/to/sepolia/data/dir --jwt-secret /path/to/jwtsecret/file`
+
+This will by default try reaching out `ethereumjs` over the endpoint `8551`. (You may customize this in conjuction with `ethereumjs`, see lodestar cli help via `--help`).
+
+You may provide `--weakSubjectivityServerUrl` (with a synced `sepolia` beacon node endpoint as arg value) and  `--weakSubjectivitySyncLatest` to start directly off the head on the `sepolia` beacon chain, possibly triggering (backfill) beacon sync on ethereumjs. 
```

Comment:
Just to double-check here: are these options still working/up-to-date?

Reply:
yes, these options work! we need a working sepolia synced beacon node exposing debug endpoints to download state, but i don't have one handy hosted by lodestar for sepolia